src/slider/slider.scss

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-s+$iui-xxs;
+
+@mixin iui-slider-component-container {
+  display: inline-flex;
+
+  span.iui-left,
+  span.iui-right {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: 4px;
+    }
+  }
+  
+  span.iui-left {
+    margin-right: $iui-s;
+    text-align: right;
+  }
+
+  span.iui-right {
+    margin-left: $iui-s;
+  }
+}
+
+@mixin iui-slider-container {
+  display: flex;
+  position: relative;
+  z-index: 1; // above ticks
+  width: 100%;
```

Comment:
As I mentioned above, this value is not really correct.. cause in same container there are left and right elements, so this one just cannot take 100% of whole container..
It works, cause container has flex applied..so kinda hacky

Reply:
I no longer use display flex. Not sure what is "hacky" please explain.

Reply:
Flex is still being used on `iui-slider-component-container`, which is the parent of `iui-slider-container`.
So `iui-slider-component-container` has `iui-left` and `iui-right` in children too. So all I am saying that practically `iui-slider-container` cannot take 100% width of the parent, cause parent has also 2 more elements which it needs to fit. Just now it works because of flex, so.. Better here would be to just apply `flex-grow: 1` on `iui-slider-container` so it has same outcome.

Reply:
Changed